server/src/test/scala/org/scassandra/server/actors/ConnectionHandlerTest.scala

```diff
@@ -303,4 +320,9 @@ class ConnectionHandlerTest extends TestKit(ActorSystem("Test")) with Matchers w
     testActorRef ! Received(ByteString(startupMessage.toArray))
   }
 
+  private def clear(): Unit = {
```

Comment:
This must just be handling any case where we get messages in between tests?  When would this happen?